components/Home/Header/Logo.jsx

```diff
@@ -1,5 +1,5 @@
 import Image from "next/image";
-import { Images } from "../../../constants/constants";
+import { Images } from "constants/index";
```

Comment:
no need of `index` here

Reply:
I tried it but not working , so I chosen this method